src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -400,6 +401,7 @@ def load_command_table(self, _):
         g.custom_command('update-instances', 'update_vmss_instances', supports_no_wait=True)
         g.wait_command('wait', getter_name='get_vmss', getter_type=compute_custom)
         g.command('get-os-upgrade-history', 'get_os_upgrade_history', min_api='2018-10-01')
+        g.custom_command('set-orchestration-service-state', 'set_orchestration_service_state', supports_no_wait=True)
```

Comment:
is this a long running operation?

Reply:
Yes, it's a long running operation. And vmss already have a wait command.